ishoborabyose/redux-todo
# Feat: Redux-todo

## Description
This pull request adds a to-do component to the  Redux application. With this feature, users can type todo,  users can delete todo, users can navigate through a page to view additional todos.
## Changes Made

- Added new currentPage and postsPerPage state variables to the Posts component's Redux store
- Modified the fetchPosts action creator to accept an optional page parameter, which retrieves only the posts for the specified page
- Added a new Pagination component, which displays page numbers and allows the user to navigate between pages
- Updated the Posts component to display only the posts for the current page, and to render the Pagination component

## How to Test

To test this feature:

- Open the index page in the Redux application
- Observe that only a certain number of todo are added on the page and that todo component is displayed on the page
- Click on one of the todos delete icons to delete
- Click on one of the todo checkboxes to mark the done line through 

## Related Issues

This feature is for creating a to-do application using redux technology.

## Screenshots

![image](https://user-images.githubusercontent.com/52440263/228255190-87b77b62-3457-4aee-9315-147caf2e9735.png)
